README.md

```diff
@@ -123,6 +124,7 @@ You can adjust behavior of the Docker container by specifying these Docker-speci
 | `LITESTREAM_ENDPOINT`          | Litestream-compatible cloud storage endpoint where Litestream should replicate data.                  |
 | `LITESTREAM_ACCESS_KEY_ID`     | Litestream-compatible cloud storage access key ID to the bucket where you want to replicate data.     |
 | `LITESTREAM_SECRET_ACCESS_KEY` | Litestream-compatible cloud storage secret access key to the bucket where you want to replicate data. |
+| `LITESTREAM_RETENTION`         | The amount of time Litestream snapshots & WAL files will be kept. (default 72h)                       |
```

Comment:
Can we say "(defaults to 72h)." to match the conventions above?

(add 'to' and period after parens)